yarn.lock

```diff
@@ -1734,9 +1734,9 @@ map-age-cleaner@^0.1.3:
     p-defer "^1.0.0"
 
 marked@^4.0.19:
-  version "4.1.1"
-  resolved "https://registry.yarnpkg.com/marked/-/marked-4.1.1.tgz#2f709a4462abf65a283f2453dc1c42ab177d302e"
-  integrity sha512-0cNMnTcUJPxbA6uWmCmjWz4NJRe/0Xfk2NhXCUHjew9qJzFN20krFnsUe7QynwqOwa5m1fZ4UDg0ycKFVC0ccw==
+  version "4.3.0"
```

Comment:
Hmm. I may have re-generated the lock file at some point. Should I not have done that? I still get the same lock-file when re-generating it now.

Reply:
You get a different lock file because dependencies of dependencies are fetched with the latest versions, but it shouldn't be part of a PR, I'm pushing it through so we can review faster

Reply:
Gotcha. Thanks for the explanation